tests/test_utility.py

```diff
@@ -62,43 +74,54 @@ def test_parse_nodes_as_dict(monkeypatch, set_nodes, expected_nodes):
     ) == sorted(expected_nodes)
 
 
+def test_recognized_query_nodes_do_not_raise_error(monkeypatch):
+    """Test that node URLs found in the federation node index does not raise an error."""
+    monkeypatch.setattr(
+        util,
+        "FEDERATION_NODES",
+        {
+            "https://firstknownnode.org/": "My First Node",
+            "https://secondknownnode.org/": "My Second Node",
+        },
+    )
+
+    util.check_nodes_are_recognized(["https://firstknownnode.org/"])
```

Comment:
this test seems to be missing an assert

Reply:
`check_nodes_are_recognized()` will raise an error if there are any unrecognized nodes (which would cause the test to fail), so as long as there are no errors this test will pass!